src/libPMacc/include/particles/tasks/ParticleFactory.hpp

```diff
@@ -23,7 +23,7 @@
 
 #pragma once
 
-#include "Environment.hpp"
+#include "Environment.def"
```

Comment:
not sure if the according `.tpp` file should then include the `Environment.hpp` because it [does use it](https://github.com/Flamefire/picongpu/blob/8bc571e93d1515b5c42be4a32858bc5551ddfb5e/src/libPMacc/include/particles/tasks/ParticleFactory.tpp#L43) and won't work with a forward declaration.